lib/rules/ruleWarningMessages.js

```diff
@@ -0,0 +1,23 @@
+import entries from 'object.entries';
```

Comment:
doesn't `Object.entries` work? I guess for that we do not need a external library